stephan-livera-podcast/2018-07-28-Vijay Boyapati-Stephan Livera.md

```diff
@@ -0,0 +1,216 @@
+---
+title: How Bitcoin ‘mentally captures’ people.
+transcript_by: Dereje Hinsermu
```

Comment:
```suggestion
transcript_by: Stephan Livera
```

If this is taken from https://stephanlivera.com/, then it should be Stephan's name.